zerver/decorator.py

```diff
@@ -898,7 +898,7 @@ def _wrapped_view_func(
         *args: ParamT.args,
         **kwargs: ParamT.kwargs,
     ) -> HttpResponse:
-        if not request.user.is_authenticated:
+        if not request.user.is_authenticated:  # nocoverage
```

Comment:
(Maybe some of the lost tests can be rewritten against a different endpoint?)

Reply:
I think `webathena_kerberos_login` is the only remaining endpoint with this decorator? We might want to just change that to use the normal API decorators.

Reply:
Oh, hmm, I'm wrong -- we also use this endpoint in other decorators -- so we should actually add back that test coverage. Should have noticed this before merging, sigh.

Reply:
- #29504